### PR TITLE
Document "christophwurst/nextcloud replacement" and "Disabled keyboar…

### PR DESCRIPTION
**Pull request by nickvergessen**
…d shortcuts" for 25

grafik

### CODE REVIEW

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst [hunk 1]
```diff
@@ -28,12 +28,29 @@ See `Github issue <https://github.com/nextcloud/server/issues/32060>`_.
 Front-end changes
 -----------------
 
-tbd
+Disabled keyboard shortcuts
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:31]: This change can be mentioned here but should find a place in the regular documentation. Maybe in the front-end parts or a11y section if we have one.

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst [hunk 2]
```diff
@@ -28,12 +28,29 @@ See `Github issue <https://github.com/nextcloud/server/issues/32060>`_.
...
+If that is the case, no additional shortcuts shall be registered by any app. Only ``space`` to toggle checkboxes and
+``enter`` to submit the currently active buttons/links are okay to be used.
+See `Github issue <https://github.com/nextcloud/server/pull/34081>`_.
 
 Back-end changes
 ----------------
 
-tbd
+christophwurst/nextcloud replaced
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:46]:
```suggestion
``christophwurst/nextcloud`` replaced
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst [hunk 3]
```diff
@@ -28,12 +28,29 @@ See `Github issue <https://github.com/nextcloud/server/issues/32060>`_.
...
+
+The `christophwurst/nextcloud <https://packagist.org/packages/christophwurst/nextcloud>`_ composer package was replaced
+with a now Nextcloud owned `nextcloud/ocp <https://packagist.org/packages/nextcloud/ocp>`_ package. The content is the
+same and all older versions were generated, so you can transition right away no matter which versions you support.
+We also have a GitHub Actions cron job template available at
+`https://github.com/nextcloud/.github/blob/master/workflow-templates/update-nextcloud-ocp.yml <https://github.com/nextcloud/.github/blob/master/workflow-templates/update-nextcloud-ocp.yml>`_
+which updates the packages every sunday, so you can make sure your app is still compatible with the latest OCP.
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:53]: this info is not relevant for the upgrade and can go into the CI docs

**Review by ChristophWurst — COMMENTED** [review]
I would suggest to split the information that is relevant for updating an app vs regular app documentation

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst (hunk 1 above)
Comment (nickvergessen) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:31]: Already did that back then: https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/js.html#keyboard-shortcuts

Comment (ChristophWurst) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:31]: Then I suggest to create a ref and link to them so one can jump into the details

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst (hunk 3 above)
Comment (nickvergessen) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:53]: Will update CI docs in a second PR to not block this file or get conflicts when others document their 25 stuff

**Comment by nickvergessen** [comment]
Incorporated the changes and updated the screenshot

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst (hunk 3 above)
Comment (nickvergessen) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst]: Done in https://github.com/nextcloud/documentation/pull/9299

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst (hunk 1 above)
Comment (nickvergessen) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst:31]: grafik


Done

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**